spell/spell-crafter-mainnet-workflow.md

```diff
@@ -4,11 +4,35 @@
 
 Repo: https://github.com/makerdao/spells-mainnet
 
-### [Governance Cadence Improvement](https://forum.makerdao.com/t/governance-cadence-improvement/14972)
+### Spell coordination schedule
 
-![](https://ipfs.io/ipfs/QmUqCvy7c8Qmzn7yZ6D3353wTqCZ3VDAwQKYB37pJ2BjXb)
+| Responsible | Stage                                              | Deadline                         |
+|:------------|:---------------------------------------------------|:---------------------------------|
+| Governance  | Exec Sheet is created                              | 16:00 UTC Week 1 Tuesday         |
+| All         | Agreement is reached on the content and roles      | 16:00-16:30 UTC Week 1 Tuesday   |
+| Crafter     | Spell is cleaned up (for external contributions)   | 12:00 UTC Week 1 Wednesday       |
+| External    | External code is contributed via PR (if required)  | 16:00 UTC Week 1 Thursday        |
+| Governance  | Exec Sheet is finalised and fully confirmed        | 16:00 UTC Week 1 Thursday        |
+| Crafter     | Spell is crafted (without the Exec Hash)           | 16:00 UTC Week 1 Friday          |
+| Reviewers   | Spell code is reviewed (against the Exec Sheet)    | 16:00 UTC Week 2 Monday          |
+| Governance  | Exec Doc is merged                                 | 16:00 UTC Week 2 Monday          |
+| Crafter     | Spell code review is addressed, Exec Hash is added | 12:00 UTC Week 2 Tuesday         |
+| Reviewers   | Spell code is reviewed (against the Exec Doc)      | 16:00 UTC Week 2 Tuesday         |
+| Crafter     | Spell is deployed, Testnet is created              | 12:00 UTC Week 2 Wednesday       |
+| Reviewers   | Spell deployment is approved                       | 16:00 UTC Week 2 Wednesday       |
+| Crafter     | Handover is published                              | 16:00-16:30 UTC Week 2 Wednesday |
```

Comment:
For clarity, I believe this could be phrased as:
```suggestion
| Crafter     | Spell address is published                              | 16:00-16:30 UTC Week 2 Wednesday |
```

Same for the items below.

Other than that, changes look good.

Reply:
Thanks, updated the wording